scripts/survival_migration/survivalDataMigration.py

```diff
@@ -0,0 +1,197 @@
+import csv
+import pandas as pd
+import numpy as np
+import os
+import sys
+import argparse
+import shutil
+
+PARENT_PARSER_NAME = 'survival data migration tool'
+NULL_VALUES = ["[not applicable]", "[not available]", "[pending]", "[discrepancy]", "[completed]", "[null]", "", "na"]
+DEFAULT_MAPPING_FILE_PATH = os.getcwd() + '/survivalStatusVocabularies.txt'
```

Comment:
I still don't understand why you have to "hardcode" this name into the script.  Why not always require a vocabulary file?